packages/ai-jsx/src/core/inline.tsx

```diff
@@ -6,10 +6,10 @@ type InlineChild = Node | InlineFn;
 type InlineChildren = InlineChild | InlineChildren[];
 
 /**
- * Use this to do a completion where you have access to the previously-rendered content.
+ * The Inline component renders each of its children in sequence, optionally passing the previously rendered content to the next child.
```

Comment:
I think it would be nice to link to the tutorial section that talks about `Inline`.